lnplus/docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: $APP_LNPLUS_IP
+      APP_PORT: $APP_LNPLUS_PORT
+  web:
+    image: lightningnetworkplus/lnpclient:v0.1.11@sha256:fab527c4c0d597e78e18a5ece0002d590b311f910249330ab2447b8f36bc76f2
+    restart: on-failure
+    stop_grace_period: 1m
+    volumes:
+      - ${APP_LIGHTNING_NODE_DATA_DIR}:/lnd:ro
+    environment:
+      RAILS_ENVIRONMENT: $APP_LNPLUS_RAILS_ENVIRONMENT
```

Comment:
Unless it's absolutely neccessary for other apps to know your rails environment, we could just hardcode `production` here?